apps/web/src/hooks/useExperimentalFeatureEnabled.ts

```diff
@@ -0,0 +1,12 @@
+import { EXPERIMENTAL_FEATURES } from 'config/experminetalFeatures'
+import Cookies from 'js-cookie'
+import { useState } from 'react'
+
+const useExperimentalFeatureEnabled = (featureFlag: EXPERIMENTAL_FEATURES) => {
+  const isCookieInFeatureFlags = () => Cookies.get(`ctx-${featureFlag}`)?.toString() === 'true'
+  const [cookieValue] = useState<boolean>(isCookieInFeatureFlags())
+
+  return { featureEnabled: cookieValue }
+}
+
+export { useExperimentalFeatureEnabled }
```

Comment:
let's add export to the function directly
```suggestion
export const useExperimentalFeatureEnabled = (featureFlag: EXPERIMENTAL_FEATURES) => {
```